code/cgame/cg_draw.c

```diff
@@ -1323,7 +1323,7 @@ static void CG_DrawDisconnect(void) {
 		x = (640 - 48 - 94);
 	} else if (cgs.gametype == GT_BALLOON) {
 		x = (640 - 48 - 64);
-	} else if (cgs.gametype == GT_CTF) {
+	} else if ((cgs.gametype == GT_CTF) || (cgs.gametype == GT_1FCTF)) {
```

Comment:
`if (cgs.gametype == GT_CTF || cgs.gametype == GT_1FCTF)` this is enough - no extra `()` needed